docs/SENTENCES.md

```diff
@@ -2,20 +2,28 @@
 
+## In a few words...
+📝 [Sentence collector](https://github.com/common-voice/sentence-collector) is a tool for contributors to upload public domain sentences through a website (the [Sentence Collector website](https://commonvoice.mozilla.org/sentence-collector/)) which then can get reviewed and are exported to the [Common Voice database](https://github.com/common-voice/common-voice/tree/main/server/data). Once imported into the Common Voice website, they will show up for contributors to read out aloud. This is a good place to start for newcomers to this project.
+
+📘 Contributors who want to [bulk upload](https://en.wikipedia.org/wiki/Bulk_insert) thousands of sentences, like for books, should check out the [Bulk Submission](https://github.com/common-voice/common-voice/blob/main/docs/SENTENCES.md#bulk-submission) guidelines below. There is no dedicated repository for this.
+
+🖥️ For automatic extraction of data sources, the [Sentence Extractor](https://github.com/Common-Voice/cv-sentence-extractor) is dedicated for extracting from sources such as Wikipedia, Wikisource or raw files.
+
 ## Sentence Collector
 
-The [Sentence Collector](https://commonvoice.mozilla.org/sentence-collector/) is a tool for crowdsourcing sentences for Common Voice. You can either:
+The [Sentence Collector](https://commonvoice.mozilla.org/sentence-collector/) is a website for crowdsourcing sentences for Common Voice. You can either:
 
-- Add sentences for your language
+- Add sentences for your language 
 - Validate sentences that other contributors have added
 
-Each sentence requires at least two upvotes to be considered valid.
+Each sentence requires at least two upvotes from human validation to be considered valid.
 
-At regular intervals, validated sentences from the Sentence Collector will be exported and added to the Common Voice database. These will be available on the site next time there's a release.
+Every week, validated sentences from the Sentence Collector will be exported and added to the Common Voice repository, and will be available at the next release of the Common Voice website. 
 
+For more detailled explanations, see the [ReadMe file](https://github.com/common-voice/sentence-collector/blob/main/README.md) of the Sentence Collector.
 ## Automatic extraction
 
-The Sentence Extractor is a tool that can scrape public domain data sources for sentences. Right now, the only source we have configured is Wikipedia. Please [see this post](https://discourse.mozilla.org/t/sentence-extractor-current-status-and-workflow-summary/62332) for detailed guidance on how to use the sentence extractor.
+The Sentence Extractor is a tool that can scrape public domain data sources for sentences. There are multiple sources integrated into the Sentence Extractor, such as Wikipedia and Wikisource. Please [see this post](https://discourse.mozilla.org/t/sentence-extractor-current-status-and-workflow-summary/62332) for detailed guidance on how to use the Sentence Extractor.
```

Comment:
Haven't noticed this before, but probably wouldn't be bad to link to it here. Even if it's already linked further above.

```suggestion
The [Sentence Extractor](https://github.com/Common-Voice/cv-sentence-extractor) is a tool that can scrape public domain data sources for sentences. There are multiple sources integrated into the Sentence Extractor, such as Wikipedia and Wikisource. Please [see this post](https://discourse.mozilla.org/t/sentence-extractor-current-status-and-workflow-summary/62332) for detailed guidance on how to use the Sentence Extractor.
```